lstchain/reco/r0_to_dl1.py

```diff
@@ -267,6 +268,44 @@ def get_dl1(
     return dl1_container
 
 
+def apply_lh_fit(
+        event,
+        dl1_container,
+        fitter
+):
+    """
+    Prepare and performs the extraction of DL1 parameters using a likelihood
+    based reconstruction method.
+
+    Parameters
+    ----------
+    event: ctapipe event container
+    dl1_container: DL1ParametersContainer
+    fitter: TimeWaveformFitter
+
+    Returns
+    -------
+    DL1LikelihoodParametersContainer
+
+    """
+    # Applied to all simulated event and all physic events
+    if dl1_container.mc_type > -1000 or (event.trigger.event_type == EventType.SUBARRAY):
```

Comment:
I honestly don't remember. If `EventType` was always filled it is probably a mistake. If not it could be because part of this was written in lstchain 0.6.3. I will do the modification in the next commit.